makefile

```diff
@@ -8,7 +8,7 @@ node_modules:
 	npm install prettier clang-format css-validator html-validate eslint eslint-config-google
 
 devprep: 
-	cp src/main/webapp/vendor/startbootstrap/index.html src/main/webapp/index.html
+	cp src/main/webapp/vendor/startbootstrap/index.jsp src/main/webapp/index.jsp
```

Comment:
do we still need to copy this file at all? can we just serve index.jsp with a Servlet that handles "/"?

Reply:
After moving index.jsp and planner.jsp out of vendor, we don't need this line. It also automatically serves index with "/"